api/v1/client/src/commonMain/kotlin/HttpClientUtils.kt

```diff
@@ -62,13 +62,6 @@ fun createOrtHttpClient(
 ): HttpClient =
     createDefaultHttpClient(json, engine).config {
         HttpResponseValidator {
-            handleResponseExceptionWithRequest { cause, request ->
```

Comment:
Again, the commit message just repeats in prose what's obvious from the diff. Please instead explain *why* this is a good thing to do.

Reply:
This PR is a draft. The content of it and the commit messages are considered a work in progress and not ready for review yet.